platform/features/plot/src/services/ExportImageService.js

```diff
@@ -63,13 +64,20 @@ define(
         function renderElement(element, type) {
             var defer = self.$q.defer(),
                 validTypes = ["png", "jpg", "jpeg"],
-                renderTimeout;
+                renderTimeout,
+                originalColor;
 
             if (validTypes.indexOf(type) === -1) {
                 self.$log.error("Invalid type requested. Try: (" + validTypes.join(",") + ")");
                 return;
             }
 
+            // Save color to be restored later
+            originalColor = element.style.backgroundColor || '';
+
+            // Defaulting to white so we can see the chart when printed
+            self.changeBackgroundColor(element, 'white');
+
             renderTimeout = self.$timeout(function () {
                 defer.reject("html2canvas timed out");
                 self.$log.warn("html2canvas timed out");
```

Comment:
We should restore the `originalColor` to the background in error cases (this render timeout, catch block below). Maybe there's a way to capture that all in a [.finally](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally) to reduce redundancy

Reply:
Oh hey, that's already being [used below](https://github.com/nasa/openmct/pull/1824/files#diff-994f63d45ed9e4e817c12967f83ae1bfR107), should be able to simply add:

```js
defer.promise.finally(self.changeBackgroundColor.bind(self, element, originalColor);
```

Note that this may (or may not) cause some issues with the spy for `changeBackgroundColor` in tests, based on previous experiences with `bind` and spies.